src/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -124,6 +124,12 @@ private int RunExecutable()
                 return result;
             }
 
+            // Add Nuget Packages Probing Path
+            var nugetPackagesRoot = _context.PackagesDirectory;
```

Comment:
If an application is under development and not published, folks will still attempt to execute it via "dotnet <app.dll>". How is the hint being passed to the muxer to actually map that to "dotnet run <app.dll>" and invoke CLI @anurse @piotrpMSFT @brthor @schellap ?